asv_bench/benchmarks/series_methods.py

```diff
@@ -15,6 +15,21 @@ def time_series_constructor_no_data_datetime_index(self):
         Series(data=None, index=self.dr)
 
 
+class series_constructor_dict_data_datetime_index(object):
+    goal_time = 0.2
+
+    def setup(self):
+        self.dr = pd.date_range(
```

Comment:
can you make this 1/10th the size (it will still show the differences), just generally don't want to have have 3 second functions. ping when you push as the rest is ok.